### PR TITLE
Bugfix #3 Fix rate limit error

### PR DESCRIPTION
**Pull request by cyrmans**
Use endpoint "/import/issues" to avoid rate limit errors
Related to issue #3

### CODE REVIEW

**Comment by gavinr** [comment]
Thanks for the PR. I'll test it out.

File: import.js [hunk 1]
```diff
@@ -33,37 +33,35 @@ const importFile = (octokit, file, values) => {
...
           // if we have a milestone column, pass that.
           if (milestoneIndex > -1 && row[milestoneIndex] !== "") {
-            sendObj.milestone = row[milestoneIndex];
+            sendObj.issue.milestone = row[milestoneIndex];
           }
 
           // if we have an assignee column, pass that.
           if (assigneeIndex > -1 && row[assigneeIndex] !== "") {
-            sendObj.assignees = row[assigneeIndex].replace(/ /g, "").split(",");
+            sendObj.issue.assignee = row[assigneeIndex].replace(/ /g, "").split(",");
```
Comment (gavinr) [inline import.js:58]: This does not work. Try importing `https://github.com/gavinr/github-csv-tools/blob/master/test/4.csv` (change the username in the csv to your username). I get error:

```
message: 'Invalid request.\n' +
        '\n' +
        `For 'properties/assignee', ["MyUserName"] is not a string or null.`,
```

**Review by gavinr — CHANGES_REQUESTED** [review]

File: import.js (hunk 1 above)
Comment (cyrmans) [reply import.js]: I fixed the assignee problem and I test with all tests files.
All are OK (noTitle.csv return "Title required by GitHub, but not found in CSV.")

**Comment by gavinr** [comment]
merged in #82. thanks

**Pull request closed without merging**